common/util_test.go

```diff
@@ -348,3 +352,99 @@ func TestRatPriceInfo(t *testing.T) {
 	assert.Nil(err)
 	assert.Zero(priceInfo.Cmp(big.NewRat(7, 2)))
 }
+
+func TestParseAccelDevices(t *testing.T) {
```

Comment:
How about creating 4 separate test functions (instead of aggregating 4 tests in one test function)? You could have something like this one.

```go
func TestParseAccessDevices_Gpu(t *testing.T) { ... }
func TestParseAccessDevices_GpuWrongDriver(t *testing.T) { ... }
func TestParseAccessDevices_GpuFailedProbing(t *testing.T) { ... }
func TestParseAccessDevices_FailedDetection(t *testing.T) { ... }
```